go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/k8gb-io/k8gb
 
-go 1.23
+go 1.23.3
 
 toolchain go1.23.4
```

Comment:
@kuritka is it expected to have different patch versions between `go` and `toolchain`?

Reply:
nice catch @ytsarev, while toolchain is same as go version, I can remove toolchain completelly and make live easier.